com.github.virtual_puppet_project.vpuppr.yml

```diff
@@ -0,0 +1,47 @@
+id: com.github.virtual_puppet_project.vpuppr
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: OpenSeeFaceGD.x86_64
+finish-args:
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--share=network" # to set up the deps for openseeface
+    - "--device=all"
+    - "--socket=pulseaudio"
+    - "--filesystem=home:ro"
+modules:
+
+  # --- Puppeteer ---
+  - name: puppeteer
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/{resources,OpenSeeFaceFolder}
+      - install -Dm 644 -t /app/share/applications com.github.virtual_puppet_project.vpuppr.desktop
+      - install -Dm 644 com.github.virtual_puppet_project.vpuppr-128.png /app/share/icons/hicolor/128x128/apps/com.github.virtual_puppet_project.vpuppr.png
+      - mkdir -p /app/share/icons/128x128/apps/
+
+      - mv * /app/bin/
+      - mv /app/bin/gui /app/bin/licenses /app/bin/scripts /app/bin/translations /app/bin/resources/
+      - mv /app/bin/OpenSeeFace /app/bin/OpenSeeFaceFolder/
```

Comment:
the appdata file is still NOT installed. and it should be listed in the sources too.

Reply:
Do you mean `com.github.virtual_puppet_project.vpuppr.metainfo.xml`? If you do, how would I go about installing it?

Reply:
https://github.com/flathub/flathub/wiki/AppData-Guidelines#path-and-filename

Reply:
I've added the file, with steps to add it to the appropriate directories. However, it says the .xml file is not valid, despite the validation instructions you linked saying it is.